### PR TITLE
Wording corrections

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix a few wording for product page
| Type?             | bug fix
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | CI green
| Possible impacts? | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23758)

There are a few PRs which were merged while still having the label Waiting for wording:

https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+label%3A%22Waiting+for+wording%22+milestone%3A1.7.8.0

Some wording need to be confirmed or improved here:
https://github.com/PrestaShop/PrestaShop/pull/24056

The purpose of this PR is to fix the wording that require it

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by jolelievre** [comment]
Sorry I forgot to declare this one as WIP, we have a few other fixes to integrate

**Comment by Julievrz** [comment]
> There are a few PRs which were merged while still having the label Waiting for wording:
> 
> https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+label%3A%22Waiting+for+wording%22+milestone%3A1.7.8.0
> 

Wording check WIP: 

- [x] https://github.com/PrestaShop/PrestaShop/pull/24049 
- [x] https://github.com/PrestaShop/PrestaShop/pull/23894 (fixed in a new PR)
- [x] https://github.com/PrestaShop/PrestaShop/pull/23689 
- [x] https://github.com/PrestaShop/PrestaShop/pull/23687 => @jolelievre I added a review to this PR with some wordings changes. 
- [x] https://github.com/PrestaShop/PrestaShop/pull/23667 
- [x] https://github.com/PrestaShop/PrestaShop/pull/23616 => a small change here that I had missed
- [x] https://github.com/PrestaShop/PrestaShop/pull/22519 
- [x] https://github.com/PrestaShop/PrestaShop/pull/19289 
- [x] https://github.com/PrestaShop/PrestaShop/pull/17703 
- [x] https://github.com/PrestaShop/PrestaShop/pull/16872 
- [x] https://github.com/PrestaShop/PrestaShop/pull/16553

**Comment by jolelievre** [comment]
@Julievrz I just fixed the wording you commented 😉
Thank you

**Review by matks — DISMISSED** [review]

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php [hunk 1]
```diff
@@ -61,17 +61,17 @@ public function configureOptions(OptionsResolver $resolver)
...
             'translations' => [
                 'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
-                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
-                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings updated', 'Admin.Global'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Catalog.Notification'),
                 'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
                 'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
-                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('Choosing another image as the cover will automatically uncheck this box.', 'Admin.Actions'),
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php]:
```suggestion
                'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Actions'),
```
I slightly modified this one so it is even more consistent with the rest of the wording. :) Apart from this everything is good, thank you!

**Review by Progi1984 — DISMISSED** [review]

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php [hunk 2]
```diff
@@ -61,17 +61,17 @@ public function configureOptions(OptionsResolver $resolver)
...
             'translations' => [
                 'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
-                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
-                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings updated', 'Admin.Global'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Catalog.Notification'),
                 'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
                 'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
-                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('Using another image as the cover will automatically uncheck this box.', 'Admin.Actions'),
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php]:
```suggestion
                'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Actions'),
```

File: src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php [hunk 3]
```diff
@@ -61,17 +61,17 @@ public function configureOptions(OptionsResolver $resolver)
...
             'translations' => [
                 'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
-                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
-                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings updated', 'Admin.Global'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Catalog.Notification'),
                 'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
                 'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
-                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Actions'),
```
Comment (Julievrz) [inline src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php]:
```suggestion
                'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Catalog.Help'),
```

**Comment by Julievrz** [comment]
There is just one domain change left and then the wording will be ✔️

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @jolelievre ,

Wording is OK !

QA :heavy_check_mark: 

Thanks !

**Comment by Progi1984** [comment]
Thanks @jolelievre & @Robin-Fischer-PS

**Pull request merged**